config/routes.rb

```diff
@@ -72,7 +72,7 @@
   constraints SplitCheckoutConstraint.new do
     get '/checkout', to: 'split_checkout#edit'
 
-    constraints step: /(details|payment|summary)/ do
+    constraints step: /(enter-your-details|payment-method|order-summary)/ do
```

Comment:
Not sure of this one @Matt-Yorkley .
URLs are defined in https://github.com/openfoodfoundation/openfoodnetwork/issues/7409 and must match: 
/checkout/enter-your-details
/checkout/payment-method
/checkout/order-summary
But actually I don't know if I took the right path (haha) to do it.

Reply:
I think it would be much better if we kept the simpler no-hyphenated versions, as we need to use them programmatically as Ruby symbols, which generally don't like hyphens...